demos/JWST/S6_template.ecf

```diff
@@ -30,6 +30,7 @@ model_spectrum  None # Path to the model spectrum relative to topdir. Set to Non
 model_x_unit    um # Options include any measurement of light included in astropy.units.spectral (e.g. um, nm, Hz, etc.)
 model_y_unit    Rp/Rs # Options include Rp/Rs, (Rp/Rs)^2, Fp/Fs
 model_y_scalar  1 # Indicate whether model y-values have already been scaled (e.g. write 1e6 if model_spectrum is in ppm)
+model_zorder    0 # The zorder of the model on the plot (0 for beneath the data, np.inf for above the data)
```

Comment:
np.inf seems unnecessary.  What's the zorder of the data?  If 1, for example, the user should be instructed to set model_zorder to >1 for above the data.

Reply:
This was based on my past experience where the errorbars end up being at a different zorder than what I'd expected, so I've historically done np.inf just to make sure it works. I agree it's unnecessary though - I'll look into a more elegant solution